### ISSUE TITLE
2.1.1 Promotional Cart Rule Conditions issue

### ISSUE THREAD

**Issue by dapdx** [magento/magento2#6911]
In Magento 2.1.1 there is an issue with the Promotional Cart Rules conditions. The top Apply run only if the following conditions are met works fine. But below when you go to add the conditions to apply the rule to specific products the field does not retain the selection.
### Preconditions

1. Magento 2.1.1, PHP 7
2. 
### Steps to reproduce

1. Create Promotional Cart Price Rule
2. Go down to "Apply the rule only to cart items matching the following conditions"
3. Choose Category
4. Choose a category in the drop down menu
5. Hit the green Check mark to apply the selection
6. The selection should be save, but instead just shows "..."
### Expected result

1. The selection should show "Category is 11"
### Actual result

1. Shows "Category is ..."

**Comment by SerhiyShkolyarenko** [comment]
@dapdx thank you for reporting!
AFIU you mean "Actions" tab on step 2. I tried these steps on the latest develop branch and I see "Category is 4" after step 5. Do you have the same result on the latest develop?

**Comment by dapdx** [comment]
Yah, I've upgraded to 2.1.2 and still have this issue. If I know the category number I can enter it and it works, but if I use the selector box, which brings up the list of categories and I select the category I want, it doesn't save. I think it's an issue with the selector function.

**Comment by SerhiyShkolyarenko** [comment]
reproduced on v. 2.1.2, but it's fixed in develop branch.

**Comment by Ctucker9233** [comment]
@SerhiyShkolyarenko is there a commit reference?

**Comment by SerhiyShkolyarenko** [comment]
It was fixed in MAGETWO-55184. Commits: 911ec93 and a86eda3.

**Comment by SerhiyShkolyarenko** [comment]
closing as duplicate of #5526

**Comment by Ctucker9233** [comment]
@SerhiyShkolyarenko Will MAGETWO-55184 be included in 2.1.3?

**Comment by SerhiyShkolyarenko** [comment]
Yeah, it seems to be. Internal issue for backporting to 2.1 is MAGETWO-58321.

**Comment by Ctucker9233** [comment]
@SerhiyShkolyarenko Ok, I'll wait and see.

**Comment by SerhiyShkolyarenko** [comment]
@Ctucker9233 sorry, it was wrong info. MAGETWO-55184 IS devilered to 2.1.3 branch too. So, you'll see it soon.

**Comment by Ctucker9233** [comment]
@SerhiyShkolyarenko Excellent.
